explorer/src/routes/transactions/transaction/+page.svelte

```diff
@@ -20,7 +20,7 @@
     getTransaction();
   };
 
-  onNetworkChange(updateData);
+  onMount(updateData);
```

Comment:
Similar to the `block` route. The `updateData` fn can be removed and `getTransaction` can be directly called in the `onMount` hook.